pkgs/development/python-modules/e3-core/default.nix

```diff
@@ -0,0 +1,62 @@
+{ buildPythonPackage
+, colorama
+, coverage
+, distro
+, fetchFromGitHub
+, httpretty
+, lib
+, mock
+, psutil
+, pytest
+, pytest-socket
+, python-dateutil
+, pyyaml
+, requests
+, requests-toolbelt
+, stdenv
```

Comment:
```suggestion
, setuptools
, stdenv
```

Reply:
`stdenv.isLinux` is used, but I believe everything else is addressed - Thanks so much!

Reply:
The suggestion UI in github will always put the marked text in red. Note the diff both removes and adds stdenv. ;)